tests/flow/test_transpose_configurations.py

```diff
@@ -0,0 +1,152 @@
+import os
+import sys
+from RLTest import Env
+from redisgraph import Graph, Node, Edge
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+from base import FlowTestsBase
+
+GRAPH_ID = "G"
+default_graph = None
+
+class testTransposeConfigurations(FlowTestsBase):
+    def setUp(self):
+        # At the start of each test, instantiate a server with default configurations and build the graph.
+        global default_graph
+        self.env = Env()
+        self.env.cmd('set', 'foo', 'bar')
+        redis_con = self.env.getConnection()
+        default_graph = Graph(GRAPH_ID, redis_con)
+        self.populate_graph(default_graph)
+
+    def populate_graph(self, graph):
+        # Construct a graph with the form:
+        # (v1)-[:E]->(v2)-[:E]->(v3), (v1)-[:E]->(v3)
+        node_props = ['v1', 'v2', 'v3']
+
+        nodes = []
+        for idx, v in enumerate(node_props):
+            node = Node(label="L", properties={"val": v})
+            nodes.append(node)
+            graph.add_node(node)
+
+        edge = Edge(nodes[0], "E", nodes[1])
+        graph.add_edge(edge)
+
+        edge = Edge(nodes[1], "E", nodes[2])
+        graph.add_edge(edge)
+
+        edge = Edge(nodes[0], "E", nodes[2])
+        graph.add_edge(edge)
+
+        graph.flush()
+
+    # Test patterns that traverse 1 edge.
+    def test01_simple_traversal(self):
```

Comment:
Yeah, this is only a test against future regressions rather than current misbehaviors. As I don't want to expose any of the transpose logic to users, it was the best I could come up with!